pwndbg/ida.py

```diff
@@ -72,13 +72,13 @@ def init_ida_rpc_client() -> None:
     try:
         _ida.here()
         print(message.success("Pwndbg successfully connected to Ida Pro xmlrpc: %s" % addr))
-    except socket.error as e:
+    except TimeoutError:
```

Comment:
This was an interesting find by `pylint`. `TimeoutError` is a subclass of `OSError`, so it needs to be checked first, otherwise this code will never be reached.